### ISSUE TITLE
[Docs] GP Regression with Uncertain Inputs

### ISSUE THREAD

**Issue by vijay092** [cornellius-gp/gpytorch#1302]
# 📚 Documentation/Examples

Hey guys,

Could you please clarify how we could extend GP regression with uncertain inputs to a dataset with multiple features? 
i.e., when train_x_stdv  is matrix at every data point. 

I am referring to [this](https://docs.gpytorch.ai/en/v1.2.0/examples/01_Exact_GPs/GP_Regression_DistributionalKernel.html) example. 

`train_x_stdv = torch.linspace(0.03, 0.01, 20)` will now be a matrix at every datapoint. 

How is `train_x_distributional = torch.stack((train_x_mean, (train_x_stdv**2).log()), dim=1)` modified?

**Comment by island-polyu** [comment]
Hey, I wonder whether you have solved this problem, because I have the same problem.Thank you.

**Comment by gpleiss** [comment]
> train_x_stdv = torch.linspace(0.03, 0.01, 20) will now be a matrix at every datapoint.

I don't know if this kernel support matrix uncertainties, but it should probably work with a vector of uncertainties. Try setting `train_x_stdv` to a NxD matrix.
